homeassistant/components/camera/__init__.py

```diff
@@ -14,7 +14,7 @@
 import logging
 import os
 from random import SystemRandom
-from typing import Awaitable, Callable, Final, Optional, cast, final
```

Comment:
One thing I want to point out is that these were needed because of existing tests failing in https://github.com/home-assistant/core/pull/63625
When defining RtspToWebRtcProviderType  from pylint:
```
E1136: Value 'Callable' is unsubscriptable (unsubscriptable-object)
```

Was this a python 3.8 issue that was resolved?

Reply:
`unsubscriptable-object` issue was specific to 3.8
I also bumped into it a few times, but 3.8 support has been dropped so this is OK.

Reply:
That was indeed Python 3.8 specific, however, Python 3.8 support has been dropped.